tce/src/applibs/LLVMBackend/LLVMTCEIRBuilder.cc

```diff
@@ -602,6 +605,12 @@ LLVMTCEIRBuilder::buildTCECFG(llvm::MachineFunction& mf) {
             if ((j->getDesc().isCall() || isExplicitReturn(*j) ||
                  (!j->getDesc().isBranch() && instr->hasControlFlowMove())) &&
                 &(*j) != &(mbb.back())) {
+                if (operationName(*j) == "hwloop") {
```

Comment:
Do we guarantee lower case opnames from this function (I cannot recall)?

Reply:
operationName name is the one we return; i.e. https://github.com/cpc/tce/pull/132#discussion_r765719864. 
So, it is guaranteed to be lowercase.